Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -484,6 +488,8 @@ - (void)addBreadcrumbForSessionTask:(NSURLSessionTask *)sessionTask
         [hasBreadcrumb boolValue]) {
         return;
     }
+    NSDate *requestStart
+        = objc_getAssociatedObject(sessionTask, &SENTRY_NETWORK_REQUEST_START_DATE);
```

Comment:
`m`: Can't we set this directly to `nil` afterwards to reduce the memory footprint?
```
objc_setAssociatedObject(sessionTask, &SENTRY_NETWORK_REQUEST_START_DATE, nil,
        OBJC_ASSOCIATION_RETAIN_NONATOMIC);
```